legacy/address-lookup.php

```diff
@@ -15,7 +14,7 @@
         <Redirect method="GET">input-method.php?Digits=<?php echo $_REQUEST["SearchType"] . "&amp;Retry=1"; ?></Redirect>
     </Response>
      <?php
-        exit();
+        return;
```

Comment:
here we just return

Reply:
this is an inline php return

Reply:
ugh duh, I should of looked up more. thanks. :|